android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -1105,7 +1109,7 @@ public void run() {
 
     private InstabugCustomTextPlaceHolder.Key getStringToKeyConstant(String key) {
         String keyInLowerCase = key.toLowerCase();
-        switch (keyInLowerCase) {
+        switch (key) {
```

Comment:
Good catch @salmatarzi  👏 
Just Please remove the redundant line "      String keyInLowerCase = key.toLowerCase();"